src/service/gateway.proto

```diff
@@ -17,6 +20,18 @@ enum close_state {
 
 /* general */
 
+message empty {}
+
+message gateway_success_resp {
+  bytes resp = 1;
```

Comment:
remember we talked about not having details in the success resp since if there are details it should really be a specific resp message. And if it's empty then what's the difference between empty and success_resp

Reply:
yeah i did actually change these but it must be on another branch.  Will fix